src/main/java/ogc/rs/processes/tilesMetaDataOnboarding/SqlConstants.java

```diff
@@ -0,0 +1,25 @@
+package ogc.rs.processes.tilesMetaDataOnboarding;
+
+public class SqlConstants {
+    // Query to check if the collection exists in the collection_details table
+    public static final String CHECK_COLLECTION_EXISTENCE_QUERY =
+            "SELECT EXISTS (SELECT 1 FROM collections_details WHERE id = $1::UUID)";
+    // Query to get the collection type from the collection_type table
+    public static String GET_COLLECTION_TYPE_QUERY =
```

Comment:
You can use this SQL query to simplify further processing in the main codebase, i.e. in _TilesMetadataOnboardingProcess.java_- 
`select collection_id, array_agg(type) from collection_type group by collection_id having collection_id = $1::uuid`.